test/modules/CrashReporting.spec.ts

```diff
@@ -15,11 +16,17 @@ describe('CrashReporting Module', () => {
 
   it('should call the native method sendCrashReporting with JSON object and sendHandledJsCrash as a callback', () => {
     const error = new TypeError('Invalid type');
-    CrashReporting.reportError(error);
+    const userAttribute: Map<string, string> | null = null;
```

Comment:
`Map<string, string>` is different from object literals (and `Record<string, string>` which is an object literal with a `string` key and `string` value), so the type of `userAttributes` here doesn't match what it is intended to be.

We can't pass [JavaScript Map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map)s to native methods, but we can pass object literals, so if we actually use `Map` in a real app it'd cause a crash.